doc/modules/linear_model.rst

```diff
@@ -61,6 +63,21 @@ example, when data are collected without an experimental design.
 
    * :ref:`sphx_glr_auto_examples_linear_model_plot_ols.py`
 
+.. _non_negative_least_square:
```

Comment:
Nit

```suggestion
.. _non_negative_least_squares:
```

Though is this ref used anywhere?

Reply:
Indeed, this is not used. Should I remove it?

Reply:
yes, thanks